src/components/config-editor/index.tsx

```diff
@@ -10,6 +10,7 @@ import './config-editor.css';
 class ConfigEditor extends React.PureComponent<any, any> {
   public handleEditorChange = spec => {
     this.props.setConfigEditorString(spec);
+    this.props.setThemeName('custom');
```

Comment:
please rename

Reply:
What should be renamed?

Reply:
Hmm, I guess we can call it theme here. Never mind.